src/main/java/frc/robot/classes/Auton.java

```diff
@@ -179,82 +204,115 @@ private Command getAutonSequence(){
             case ONESCORECHARGE1:
                 return
                     new SequentialCommandGroup(
-                        resetOdometry("1ScoreCharge-1"),
-                        highScoreSequence(),
-                        swerveSubsystem.followTrajectory("1ScoreCharge-1", getTrajectory("1ScoreCharge-1"))
-                        .deadlineWith(armSubsystem.stowArmParallel()),
-                        swerveSubsystem.assistedBalance(true)
+                        resetOdometry("1ScoreCharge2-1"),
+                        highTransitionSequenceCone(),
+                        new StartEndCommand(() -> swerveSubsystem.setModuleStates(swerveSubsystem.convertToModuleStates(0, -0.1, 0)), () -> swerveSubsystem.stopModules()).withTimeout(0.5),
```

Comment:
This would be better as a separate method that returns a move forward.  
( this is repeated below, and if you need to change it you will have to change both. )

Reply:
Or include in hightransitionSequenceCone?  Will you ever move forward two inches other than hightransitionsequence cone?

Reply:
Changed, looks good